docs/books/modules/user-guide/listening-client-connections.adoc

```diff
@@ -59,6 +60,7 @@ listener {
 +
 For more information, see xref:understanding-connection-failover-{context}[].
 --
+`multiTenant` (optional):: If true, apply multi-tenancy to endpoints connected at this listener.  The address space is defined by the virtual host (hostname field in the Open).
```

Comment:
Afaik there isn't any other documentation for multi-tenancy yet. Adding just this note will not help those who want to learn about it (what Open?, it is not clear that stands for the AMQP Open performative).

Adding just this will not help those who want to use it and it be confusing for people who do not need this feature.

Reply:
Well, this is a property that is part of the router. And the description is the exact same description from the schema, like for all the other properties.

I agree that having a full blow explanation of what this is doing would be great. But I also think that we should at least mention that this feature exists. Otherwise it is difficult to explain other why we are using an undocumented feature.

Reply:
You're completely right. If this is merged, it would be the third copy of this same paragraph in dispatch ;)

It's not undocumented feature, even now. You can see it in the config file docs at https://qpid.apache.org/releases/qpid-dispatch-1.12.0/man/qdrouterd.conf.html#_listener that are autogenerated from the schema.  Afaik it is not a good idea to have the `listener` configuration example show every conceivable configuration option, just because it was a cool and new configuration option at some point in time.